### DISCUSSION TITLE
Social cards do not appear on Twitter

### DISCUSSION THREAD

**Discussion by teivah** [squidfunk/mkdocs-material#6016] [Q&A] [answered]
Hello,

I have configured my repo to use the social cards: https://github.com/teivah/100-go-mistakes/blob/master/mkdocs.yml#L34-L36

I also set up the `site_url`: https://github.com/teivah/100-go-mistakes/blob/master/mkdocs.yml#L2

When I build the project, it does generate the cards: https://github.com/teivah/100-go-mistakes/tree/master/site/assets/images/social

Yet, when I link a Tweet to 100go.co, it doesn't show the social card. Am I missing something, please?

**Comment by teivah** [comment, marked as answer]
I had to trigger a call to https://cards-dev.twitter.com/validator. It's working now, sorry.

**Reply by alexvoss** [reply]
No worries, glad this resolved itself!
